FHIR/fhir-package-loader
# Update dependencies

**Description:**
Add eslint config file based on requirements for updated eslint dependencies.

Remove @types/tar dependency because types are now included in the tar dependency.

Minor usage updates for tar and commander based on changes to package export.

Minor fixes for linting rules.

**Testing Instructions:**
Confirm that all tests pass. Confirm that CLI works as expected. There will be additional review and testing when the `new-fpl` branch is being pulled into `main`.

**Related Issue:**
Fixes #27
